### PR TITLE
Pydantic settings docs

### PR DESCRIPTION
**Pull request by tpdorsey**
Adds a new page for pydantic-settings.

Content is pulled from the pydantic-settings repo docs. 

https://github.com/pydantic/pydantic-settings/pull/48 will fix formatting of the imported page. The plugin expects the first line to be a heading and drops it, so adding a blank line avoids losing the first line of the text.

## Checklist

* [ ] Unit tests for the changes exist
* [ ] Tests pass on CI and coverage remains at 100%
* [ ] Documentation reflects the changes where applicable
* [ ] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)
* [ ] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

Selected Reviewer: @samuelcolvin

### CODE REVIEW

**Comment by tpdorsey** [comment]
please review

**Comment by samuelcolvin** [comment]
output:

```
19:50:57.967 |  -- | --
19:50:57.967 | 🎉 All complete!
19:50:57.969 |  
19:50:58.737 | + pdm run python -c 'import docs.plugins.main'
19:51:00.581 | + pdm run mkdocs build
19:51:02.865 | INFO     -  Cleaning site directory
19:51:02.865 | INFO     -  Building documentation to directory: /opt/buildhome/repo/site
19:51:04.221 | WARNING  -  mkdocstrings_handlers: Formatting signatures requires Black to be installed.
19:51:11.897 |  
19:51:11.897 | Aborted with 1 warnings in strict mode!
19:51:12.179 | Failed: build command exited with code: 1
19:51:13.049 | Failed: error occurred while running build command
```

File: docs/usage/errors.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Below are details on common errors developers can see when working with pydantic, together
 with some suggestions on how to fix them.
 
+{% raw %}
```
Comment (samuelcolvin) [inline docs/usage/errors.md:4]: what do these "raw" tags do?

File: docs/usage/pydantic-settings.md
Comment (samuelcolvin) [inline docs/usage/pydantic-settings.md:1]: `docs/usage/pydantic_settings.md` to be consistent.

File: docs/usage/pydantic-settings.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+description: Support for loading a settings or config class from environment variables or secrets files.
+---
+
+# Pydantic Settings
```
Comment (samuelcolvin) [inline docs/usage/pydantic-settings.md:5]:
```suggestion
# Settings Management
```

**Comment by samuelcolvin** [comment]
please update.

File: docs/usage/errors.md (hunk 1 above)
Comment (tpdorsey) [reply docs/usage/errors.md:4]: Jinja2 is needed to handle the markdown embed, but as a side effect it interprets the `{#` used to specify anchor links as comments and fails. The `{% raw %}` tag fences this page and jinja effectively ignores it.

**Review by samuelcolvin — APPROVED** [review]

**Pull request merged**